app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -84,6 +89,7 @@ open class DataStore(
         // register for actions
         dispatcher.register
             .filterByType(DataStoreAction::class.java)
+            .debug("datastore actions from dispatcher")
```

Comment:
`.debug`

Reply:
still a WIP branch, so I think this kind of thing is acceptable :)